openedx/features/smart_referral/helpers.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth.models import User
+
+from lms.djangoapps.onboarding.models import UserExtendedProfile, Organization
+
+CONTACT_EMAIL_KEY = 'contact_email'
+
+
+def get_platform_contacts_and_non_platform_contacts(contacts):
+    """
+    Returns two(2) list first list belongs to contacts of users that are currently registered on our platform and
```

Comment:
This could be an approach:





"""
    Return two list of contacts
    @return first list: Contacts of users that are currently registered on our platform
    @return second list: Contacts of users that are currently not part of our platform
"""

Reply:
done